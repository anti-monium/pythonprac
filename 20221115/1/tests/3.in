@objcount
class C:
	def __del__(self):
		print('Bye')

c = C()
d, e = C(), C()
print(C.counter)
c = 100500
print(C.counter)
f = C()
del e
print(C.counter)
