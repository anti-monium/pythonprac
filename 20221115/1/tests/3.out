3
Bye
2
Bye
2
Bye
Bye
